cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/PushUtils.java

```diff
@@ -196,12 +197,14 @@ private static void pushBaloToRemote(Path baloPath) {
                                               "'ballerina build -c <module_name>' to compile and generate the balo.");
             }
     
-            Optional<Path> moduleBaloFile = Files.list(baloOutputDir)
-                    .filter(baloFile -> null != baloFile.getFileName() &&
-                                        baloFile.getFileName().toString().startsWith(moduleName + "-" +
-                                                                                     IMPLEMENTATION_VERSION))
-                    .findFirst();
-    
+            Optional<Path> moduleBaloFile;
+            try (Stream<Path> stream = Files.list(baloOutputDir)) {
```

Comment:
Instead of "stream" shall we use "fileStream". "stream" is too vague for me.

Reply:
Renamed variable.